include/baresip.h

```diff
@@ -512,6 +512,8 @@ struct list     *contact_list(const struct contacts *contacts);
 const char      *contact_str(const struct contact *c);
 const char      *contact_uri(const struct contact *c);
 const char      *contact_presence_str(enum presence_status status);
+enum sdp_dir     contact_adir(const struct contact *c);
+enum sdp_dir     contact_vdir(const struct contact *c);
```

Comment:
Perhaps these two functions can be combined into one function
that takes one extra argument `bool audio` ?

Also `au` or `audio` is more descriptive than just `a` ...

Reply:
```
void contact_get_ldir(struct call *call, enum sdp_dir *ap, enum sdp_dir *vp);
```
similar to existing `call_get_mdir()`.

We don't need return int err here.